plugin.yml

```diff
@@ -3,9 +3,6 @@ name: HideCommands
 version: 2.0.1
 main: Himbeer\HideCommands\Main
 api: 4.0.0
-mcpe-protocol:
```

Comment:
The plugin will not get accepted on Poggit when packets are used directly and the protocol isn't specified